modes/dictation_mode.talon

```diff
@@ -3,12 +3,14 @@ mode: dictation
 ^press <user.keys>$: key("{keys}")
 
 # Everything here should call auto_insert to preserve the state to correctly auto-capitalize/auto-space.
-<user.prose>: auto_insert(prose)
-new line: "\n"
-new paragraph: "\n\n"
-cap <user.word>:
-    result = user.formatted_text(word, "CAPITALIZE_FIRST_WORD")
-    auto_insert(result)
+<user.raw_prose>: auto_insert(raw_prose)
+cap: user.dictation_format_cap()
+# Hyphenated variants are for Dragon.
+(no-caps | no caps): user.dictation_format_no_caps()
+(no-space | no space): user.dictation_format_no_space()
+^cap that$: user.dictation_reformat_cap()
```

Comment:
FYI I added anchors to these commands. I don't need this in my grammar because I'm using the experimental anchor-file feature so I just move these to a separate file so they can't be chained with dictation.